manager/assets/modext/widgets/security/modx.grid.role.js

```diff
@@ -44,7 +44,7 @@ MODx.grid.Role = function(config) {
             ,sortable: true
         }]
         ,tbar: [{
-            text: _('create_new')
+            text: _('new_role')
```

Comment:
You can't rename the lexicon key here without renaming it in the lexicon files.

Reply:
This entry is on the way core/lexicon/en/default.inc.php - in 279 line. But she was not used

Reply:
Ok, thanks.